app/org/sagebionetworks/bridge/services/UserAdminServiceImpl.java

```diff
@@ -154,16 +146,4 @@ private boolean deleteUserInStudy(Study study, Account account, User user) throw
             return false;
         }
     }
-
-    private void removeAllHealthDataRecords(Study study, User user) throws BridgeServiceException {
```

Comment:
Discussed this with Alx. I had broken up the changes in order to keep the code review small. This will be implemented in the next code change. Tracked by https://sagebionetworks.jira.com/browse/BRIDGE-363